blocks/edit/prose/loc/loc-utils.js

```diff
@@ -0,0 +1,946 @@
+import {
+  DOMSerializer,
+  Fragment,
+  Slice,
+} from 'da-y-wrapper';
+import getSheet from '../../../shared/sheet.js';
+import { htmlDiff } from './htmldiff.js';
+
+// Constants
+const LOC_COLORS = {
+  UPSTREAM: 'rgba(70, 130, 180, 0.2)',
+  LOCAL: 'rgba(144, 42, 222, 0.2)',
+  DIFF: 'rgba(150, 150, 150, 0.1)',
+};
+
+const LOC_TEXT = {
+  UPSTREAM: 'Upstream',
+  LOCAL: 'Local',
+  DIFF: 'Difference',
```

Comment:
maybe have a bigger object with text and class name for each LOC.
Also why uppercasing everything ? I think it is fine to have a `config` object all lowercase, less agressive.